climakitae/data_loaders.py

```diff
@@ -3,6 +3,8 @@
 import regionmask
 import intake
 import numpy as np
+import explore # for progress bar
```

Comment:
Instead of `import explore`, as you'd do for local work, you want this to be `from .explore import progress_bar`